### DISCUSSION TITLE
[CPU Compatibility] i686 is not possible?

### DISCUSSION THREAD

**Discussion by FabZ88** [IceWhaleTech/CasaOS#701] [Ideas]
Hi,i've many old intel Atom cpu based on i686 architecture.

I'm using one of it with Portainer. Is possible to install Casa Os on this cpu?
When I try installer stops with error "unknow architecture"

Many thanks!

**Comment by kerta1n** [comment]
Yes, i386 (i686 whatever you want to call it) is not supported. I personally have a few of those lying around, and if you feel like it, you can install plain Debian on it and install raw packages that way. I wouldn't recommend using i386 for docker as support is slowly being deprecated. Personally, I use those machines as multi-room media players with a package called snapcast.
